test/ProtocolFeesImplementation.t.sol

```diff
@@ -28,7 +27,7 @@ contract ProtocolFeesTest is Test, GasSnapshot, Deployers {
 
     function setUp() public {
         protocolFees = new ProtocolFeesImplementation();
-        feeController = new ProtocolFeeControllerTest();
+        feeController = makeAddr("feeController");
```

Comment:
definitely a nit but theres now a load of `address(feeController)` all over the tests which could just be `feeController`

Reply:
good catch